Rawl_B502/de_raind_h_érenn.trig

```diff
@@ -118,7 +118,7 @@
     a irishTitle:Rí;
     irishRel:genName "Rónáin" ;
     irishRel:nomName "Rónán" .
-    
+
 #same as http://example.com/LL/eoganachta_casil.ttlDimmaDub ?- CY
```

Comment:
Could you do me a favor? Could you move the above comment above Rónáin? It seems out of place and it easy to do now so it does not drift away from the link it is attached to.

Reply:
Done